src/reference/asciidoc/aggregator.adoc

```diff
@@ -117,6 +117,7 @@ NOTE: In the interest of code simplicity and promoting best practices such as lo
 Starting with version 5.1, after processing message group, an `AbstractCorrelatingMessageHandler` performs a `MessageBuilder.popSequenceDetails()` message headers modification for the proper splitter-aggregator scenario with several nested levels.
 It is done only if the message group release result is not a message or collection of messages.
 In that case a target `MessageGroupProcessor` is responsible for the `MessageBuilder.popSequenceDetails()` call while building those messages.
+Strting with version 5.3 if sequenceDetails of the outputMessage matches with first message of group, `MessageBuilder.popSequenceDetails()` will be performed.
```

Comment:
Comma before `if`. The `outputMessage` has to be wrapped into a code or words have to be divided.

Also consider to re-phrase the rest of the doc since it confuses now. The text flow has to be smooth somehow and cover together a `Message` and non-`Message` use-cases.